components/brave_rewards/resources/android_page/reducers/promotion_reducer.ts

```diff
@@ -81,6 +83,9 @@ const promotionReducer: Reducer<Rewards.State | undefined> = (state: Rewards.Sta
       chrome.send('brave_rewards.claimPromotion', [promotionId])
       break
     }
+    case types.RESET_PROMOTION: {
```

Comment:
should this be implemented? if not we should remove action and reducer

Reply:
It didn't seem like there was anything for it do since there's no captcha state to reset. I'll remove it, unless you think it's required for something else.